OpenRA.Mods.Common/Traits/Render/LeavesTrails.cs

```diff
@@ -111,6 +111,9 @@ void ITick.Tick(Actor self)
 			if (++ticks >= cachedInterval)
```

Comment:
This uses structured programming while the code below uses early abort. IMO, it would be best if both were early abort, but that can wait until the code needs refactoring. I don't think that is important enough to block this PR.